### ISSUE TITLE
Some keys are unnecessarily quoted

### ISSUE THREAD

**Issue by tomnomnom** [tomnomnom/gron#3]
For example keys containing underscores.

Currently:
```
{ "bar_bar": "hey" } -> json["bar_bar"] = "hey";
```

But should be:
```
{ "bar_bar": "hey" } -> json.bar_bar = "hey";
```

**Comment by tomnomnom** [comment]
It would be good to fix this 'properly' by fully adhering to http://ecma-international.org/ecma-262/6.0/#sec-identifier-names

**Comment by tomnomnom** [comment]
Fixed by 29e698419abb4cb1f6d7e90b03b070005b64ce7a
